itou/templates/dashboard/includes/diagoriente_card.html

```diff
@@ -8,7 +8,7 @@
         <div class="px-3 px-lg-4 pt-3 pt-lg-4">
             <ul class="list-unstyled mb-lg-5">
                 <li class="d-flex justify-content-between align-items-center mb-3">
-                    <a href="https://plateforme.diagoriente.beta.gouv.fr"
+                    <a href="https://diagoriente.beta.gouv.fr/services/plateforme?utm_source=emploi-inclusion-{% if user.is_prescriber %}prescripteur{% else %}employeur{% endif %}"
```

Comment:
Je l'avais pas mentionné ici aussi, mais peut-être harmoniser ? Dans un sens ou dans l'autre :).
```suggestion
                    <a href="https://diagoriente.beta.gouv.fr/services/plateforme?utm_source=emploi-inclusion-{{ request.user.is_prescriber|yesno:"prescripteur,employeur" }}"
```

Reply:
Merci, j’ai voulu aller trop vite pour me dépêcher de revenir au refactor de l’import SIAE :honey_pot: